src/main/scala/chisel3/simulator/SimulatorAPI.scala

```diff
@@ -24,11 +24,13 @@ trait SimulatorAPI {
     */
   def simulateRaw[T <: RawModule](
     module:         => T,
-    chiselSettings: ChiselSettings[T] = ChiselSettings.defaultRaw[T]
+    chiselSettings: ChiselSettings[T] = ChiselSettings.defaultRaw[T],
+    enableTrace:    Boolean = false
```

Comment:
Generally, I think we do want to expose this. However, it would be better to make it part of `ChiselSettings`. I'd like to keep the options organized under `ChiselSettings` and not have a proliferation of _other_ options.